### ISSUE TITLE
Support for Opera Browser

### ISSUE THREAD

**Issue by Bluesboy** [browserpass/browserpass-extension#270]
### General information

-   Fedora 33:
-   Opera 77.0.4054.64:
-   Information about the host app:
    -   Built from source (0997aee)
-   Information about the browser extension:
    -   Installed form Chrome Extension Store

Getting "Error: Error: Specified native messaging host not found." in popup and extension settings

**Comment by erayd** [comment]
This normally means that you have not configured the host app correctly. The documentation for doing so is [here](https://github.com/browserpass/browserpass-native#configure-browsers), however note that we do not currently have a config target for Opera - you will need to configure it manually for that browser.

How did you go about configuring it?

**Comment by Bluesboy** [comment]
I just done `make install` assuming default target will be chromium therefore compatible with Opera, turns out it's not.

**Comment by maximbaz** [comment]
Yeah sadly browsers like to be unique 😅 If you do figure out proper folders for Opera, a PR adding support would be welcome 😉

**Comment by Bluesboy** [comment]
@maximbaz Unfortunately I'm new to all this stuff, can you, please, give me some directions where to start?
